CHANGELOG.md

```diff
@@ -2,6 +2,25 @@ faucet-pipeline-core version history
 ====================================
 
 
+v2.0.0
+------
+
+_2023-01-12_
+
+notable changes for end users:
+
+* added support for faucet-pipeline-images
+
+  `plugins: [require("faucet-pipeline-images")]` is no longer required within
```

Comment:
we'd need to update [the website](https://www.faucet-pipeline.org/images) as well

Reply:
Created a PR https://github.com/faucet-pipeline/docs/pull/78